snmp/metadata.csv

```diff
@@ -1143,6 +1143,106 @@ snmp.netbotz.otherPort,gauge,,,,"[NETBOTZ410-MIB] A list of other port entries.
+snmp.netscaler.svcEstablishedConn,gauge,,,,"[NS-ROOT-MIB] Total number of connections in ESTABLISHED state.",0,snmp,,
+snmp.netscaler.svcRequestRate,gauge,,,,"",0,snmp,,
+snmp.netscaler.svcRxBytesRate,gauge,,,,"",0,snmp,,
+snmp.netscaler.svcSurgeCount,gauge,,,,"[NS-ROOT-MIB] Number of requests in the surge queue.",0,snmp,,
+snmp.netscaler.svcTotalClients,count,,,,"[NS-ROOT-MIB] Total number of established client connections.",0,snmp,,
+snmp.netscaler.svcTotalPktsRecvd,count,,,,"[NS-ROOT-MIB] Total number of packets received by this service or virtual server.",0,snmp,,
+snmp.netscaler.svcTotalPktsSent,count,,,,"[NS-ROOT-MIB] Total number of packets sent.",0,snmp,,
+snmp.netscaler.svcTotalRequestBytes,count,,,,"[NS-ROOT-MIB] Total number of request bytes received on this service or virtual server.",0,snmp,,
+snmp.netscaler.svcTotalRequests,count,,,,"[NS-ROOT-MIB] Total number of requests received on this service or virtual server. (This applies to HTTP/SSL services and servers.)",0,snmp,,
+snmp.netscaler.svcTotalResponseBytes,count,,,,"[NS-ROOT-MIB] Number of response bytes received by this service or virtual server.",0,snmp,,
+snmp.netscaler.svcTotalResponses,count,,,,"[NS-ROOT-MIB] Number of responses received on this service or virtual server. (This applies to HTTP/SSL services and servers.)",0,snmp,,
+snmp.netscaler.svcTotalServers,count,,,,"[NS-ROOT-MIB] Total number of established server connections.",0,snmp,,
+snmp.netscaler.svcTxBytesRate,gauge,,,,"",0,snmp,,
+snmp.netscaler.svcgroupCount,gauge,,,,"[NS-ROOT-MIB] Number of service groups defined on this Citrix ADC.",0,snmp,,
+snmp.netscaler.svcgroupmemCount,gauge,,,,"[NS-ROOT-MIB] Number of service group members defined on this Citrix ADC.",0,snmp,,
+snmp.netscaler.sysHealthDiskAvail,gauge,,,,"[NS-ROOT-MIB] The total disk space available in MBytes.",0,snmp,,
+snmp.netscaler.sysHealthDiskPerusage,gauge,,,,"[NS-ROOT-MIB] The Percentage of the disk space used.",0,snmp,,
+snmp.netscaler.sysHealthDiskSize,gauge,,,,"[NS-ROOT-MIB] The total disk space in MBytes (includes available and used spaces also).",0,snmp,,
+snmp.netscaler.sysHealthDiskUsed,gauge,,,,"[NS-ROOT-MIB] The total disk space used in MBytes.",0,snmp,,
+snmp.netscaler.sysHighAvailabilityMode,gauge,,,,"[NS-ROOT-MIB] This shows whether Citrix ADC is in standalone mode or whether it is primary or secondary in case of failover mode.",0,snmp,,
+snmp.netscaler.syssvcCount,gauge,,,,"[NS-ROOT-MIB] Number of services configured on this Citrix ADC.",0,snmp,,
+snmp.netscaler.sysupsvcCount,gauge,,,,"[NS-ROOT-MIB] Number of configured services which are up on this Citrix ADC.",0,snmp,,
+snmp.netscaler.sysupsvcitmCount,gauge,,,,"[NS-ROOT-MIB] Number of configured service items which are up on this Citrix ADC.",0,snmp,,
+snmp.netscaler.tcpCurClientConn,gauge,,,,"[NS-ROOT-MIB] Client connections, including connections in the Opening, Established, and Closing state.",0,snmp,,
+snmp.netscaler.tcpCurClientEstablished,gauge,,,,"",0,snmp,,
```

Comment:
Looks like some metrics don't have like `tcpCurClientEstablished` and `vsvrRequestRate`. Any idea why?

Reply:
Looks like I needed to re-run the metadata script. That seems to have gotten most of them.

`tcpCurClientEstablished` is a more unique case. Looks like the name in the profile is slightly different than in the MIB. It should be `tcpCurClientConnEstablished`. It's a bit of a mouthful as a name already so I'm tempted to leave it as is but since customers would likely want to search by the name from the MIB I'll change it to match and re-run the script